pkg/volume/fc/fc_util.go

```diff
@@ -220,7 +220,7 @@ func (util *FCUtil) AttachDisk(b fcDiskMounter) (string, error) {
 	// mount it
 	globalPDPath := util.MakeGlobalPDName(*b.fcDisk)
 	noMnt, err := b.mounter.IsLikelyNotMountPoint(globalPDPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
 		return devicePath, fmt.Errorf("Heuristic determination of mount point failed:%v", err)
```

Comment:
shouldn't create the dir first? why check whether the dir is mount or not first?

Reply:
@jingxu97 
Agree. It would work.